Libplanet/Libplanet.csproj

```diff
@@ -38,7 +38,7 @@ https://docs.libplanet.io/</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
     <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>1701;1702;CS1591;NU5104</NoWarn>
```

Comment:
A pinpoint control is probably possible instead of ignoring a warning in the entire project, like:

```
<PackageReference Include="NetMQ" Version="4.0.0.207-pre" />
  <NoWarn>NU5104</NoWarn>
</PackageReference>
```

Reply:
In my env, `dotnet build` looks like it isn't affected by `<NoWarn>` in `<PackageReference>`...

Reply:
Okay, it seems due to `NU5104` is for a project that depends.